Tchap/Managers/RestClientBuilder/RestClientBuilder.swift

```diff
@@ -43,6 +43,20 @@ final class RestClientBuilder {
         }
     }
     
+    /// Create an `MXRestClient` based on a homeserver.
+    ///
+    /// - Parameters:
+    ///   - homeserver: The homeserver.
+    ///   - completion: A closure called when the operation complete. Provide the rest client when succeed.
+    func build(_ homeServer: String, completion: @escaping (MXResponse<MXRestClient>) -> Void) {
```

Comment:
To avoid conflict with `func build(from email:`. You could write `func build(fromEmail: String`and `func build(fromHomeServer: String`.

Reply:
ok done